crypt4gh/__init__.py

```diff
@@ -50,6 +50,6 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
         except (InvalidTag) as e:
             LOG.error('Exiting for %r', e)
-            print('Invalid Key or Passphrase', file=sys.stderr)
+            LOG.error('Invalid Key or Passphrase', file=sys.stderr)
```

Comment:
No.
Don't remove the `print`, it is there on purpose.

Reply:
No problem. Happy to change this back. Just curious why its required?